src/careamics/config/nm_model.py

```diff
@@ -14,43 +16,75 @@ class GaussianMixtureNMConfig(BaseModel):
-    describing the GMM noise model, with each row corresponding to one
-    parameter of each gaussian, namely [mean, standard deviation and weight].
-    Specifically, rows are organized as follows:
-    - first n_gaussian rows correspond to the means
-    - next n_gaussian rows correspond to the weights
-    - last n_gaussian rows correspond to the standard deviations
-    If `weight=None`, the weight array is initialized using the `min_signal`
-    and `max_signal` parameters."""
+    # A [3*n_gaussian, n_coeff] sized array containing the values of the weights
+    # describing the GMM noise model, with each row corresponding to one
+    # parameter of each gaussian, namely [mean, standard deviation and weight].
+    # Specifically, rows are organized as follows:
+    # - first n_gaussian rows correspond to the means
+    # - next n_gaussian rows correspond to the weights
+    # - last n_gaussian rows correspond to the standard deviations
+    # If `weight=None`, the weight array is initialized using the `min_signal`
+    # and `max_signal` parameters.
+    weight: Any = None  # TODO wtf ?
 
+    # Number of gaussians in the GMM
     n_gaussian: int = Field(default=1, ge=1)
-    """Number of gaussians in the mixture."""
 
-    n_coeff: int = Field(default=2, ge=2)
     """Number of coefficients to describe the functional relationship between gaussian
     parameters and the signal. 2 implies a linear relationship, 3 implies a quadratic
     relationship and so on."""
+    n_coeff: int = Field(default=2, ge=2)
 
+    # Minimum signal intensity expected in the image
     min_signal: float = Field(default=0.0, ge=0.0)
-    """Minimum signal intensity expected in the image."""
 
+    # Maximum signal intensity expected in the image
     max_signal: float = Field(default=1.0, ge=0.0)
-    """Maximum signal intensity expected in the image."""
 
-    min_sigma: Any = None
-    """All values of `standard deviation` below this are clamped to this value."""
+    # All values of `standard deviation` below this are clamped to this value
+    min_sigma: Any = Field(default=200.0, ge=0.0)  # TODO took from nb in pn2v
 
+    # Tolerance used in the computation of the noise model likelihood
     tol: float = Field(default=1e-10)
-    """Tolerance used in the computation of the noise model likelihood."""
+
+    @model_validator(mode="after")  # TODO isn't it the best ever fucntion name ? :)
+    def validate_path_to_pretrained_vs_data_to_train(self: Self):
+        """_summary_.
+
+        _extended_summary_
+
+        Parameters
+        ----------
+        self : Self
+            _description_
+        """
+        if self.path and (self.signal is not None or self.observation is not None):
+            raise ValueError(
+                "Either 'path' to pre-trained noise model should be"
+                "provided or both signal and observation in form of paths"
+                "or numpy arrays"
```

Comment:
I guess this error message is wrong, here you have both `path` and `signals`. Anyway, to clarify, the idea is that we only want either only `path` or only tensors at a time, isn't it?